### PR TITLE
API-8540: describe push filtering changes

### PR DESCRIPTION
**Pull request by litwinow**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-8540-push-filtering)
- [Jira](https://livechatinc.atlassian.net/browse/API-8540)

## 📓 Description

Describe changes from https://livechatinc.atlassian.net/browse/API-7352 and https://livechatinc.atlassian.net/browse/API-7282.

## ✅ Checklist (for API docs)

- Changelog ✅
- API versions ✅
- RTM ✅
- **Table:** Available methods/webhooks/pushes ✅
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
  
### Features (for the website)

- New feature

### CODE REVIEW

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 1]
```diff
@@ -11,6 +11,11 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 
 ## [v3.3] - Developer preview
 
+### RTM
+
+- The **General RTM API Request Format** was extended with optional `version` parameter. You can now call individual methods on different (higher) version than the version of your RTM connection.
+- The **General Push Format** was extended with `version` parameter. You can subscribe to pushes from different versions using new `push` parameter in **Login** ([RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#login)).
```
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/changelog/index.mdx:17]: I try to introduce the nomenclature that uses "parameter" for requests and "field" for responses, pushes and webhooks

File: content/messaging/customer-chat-api/changelog/index.mdx [hunk 2]
```diff
@@ -11,6 +11,11 @@ This document is the record of all the changes in the **Customer Chat API**, Web
 
 ## [v3.3] - Developer Preview
 
+### RTM
+
+- The **General RTM API Request Format** was extended with an optional parameter, `version`. You can now call individual methods on different (higher) versions than the one of your RTM connection.
+- The **General Push Format** was extended with the `version` field. To receive pushes from different API versions,  subscribe to them via the new, `pushes` parameter in **Login** ([RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#login)).
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/changelog/index.mdx:17]:
```suggestion
- The **General Push Format** was extended with the `version` field. To receive pushes from different API versions, subscribe to them via the new, `pushes` parameter in **Login** ([RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#login)).
```

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 3]
```diff
@@ -11,6 +11,11 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 
 ## [v3.3] - Developer preview
 
+### RTM
+
+- The **General RTM API Request Format** was extended with an optional parameter, `version`. You can now call individual methods on different (higher) versions than the one of your RTM connection.
+- The **General Push Format** was extended with the `version` field. To receive pushes from different API versions,  subscribe to them via the new, `pushes` parameter in **Login** ([RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#login)).
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:17]:
```suggestion
- The **General Push Format** was extended with the `version` field. To receive pushes from different API versions, subscribe to them via the new, `pushes` parameter in **Login** ([RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#login)).
```

File: content/messaging/customer-chat-api/changelog/index.mdx (hunk 2 above)
Comment (litwinow) [reply content/messaging/customer-chat-api/changelog/index.mdx:17]: 😅

**Review by kogierman — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]
Feel free to rebase :D

**Pull request merged**